cmd/root.go

```diff
@@ -72,7 +72,9 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	err := rootCmd.LocalFlags().Parse(os.Args)
+	lflags := rootCmd.LocalFlags()
+	lflags.ParseErrorsWhitelist.UnknownFlags = true
+	err := lflags.Parse(os.Args)
```

Comment:
Q:
Isn't this an error message we actually want?

What happens if I misspell a flag?

Reply:
This is the global parser, bound to the root command. The different commands will still report wrong flags:

```
mender-cli (men-5428) $ ./mender-cli login --blah
Configuration file not found. Continuing.
Error: unknown flag: --blah
Usage:
  mender-cli login [flags]
```